tests/testthat/test-databricks-connect.R

```diff
@@ -0,0 +1,23 @@
+context("databricks-connect")
+
+Sys.setenv(TEST_DATABRICKS_CONNECT = 'true')
```

Comment:
We should only run this test for databricks-connect instead of all different settings in sparklyr, right? Currenly the travis test fails on all different setups.

Reply:
I tried to make it run even on travis, and just use a fake databricks-connect connection, but it does not work. So I have changed it to only run on Jenkins when we are testing dbconnect.